isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/konzept/inhalt.adoc

```diff
@@ -226,6 +226,75 @@ OpenID Connect Abfragen zur Identität eines Benutzers, OpenID-Token oder Custom
 
 [[authentifizierung-und-autorisierung]]
 === Authentifizierung & Autorisierung
+Gemäß der xref:isyfact-standards-doku:referenzarchitektur:fachlich.adoc#[IsyFact-Referenzarchitektur] wird bei der Authentifizierung nach vordefinierten xref:isyfact-standards-doku:referenzarchitektur:software-technisch.adoc#prinzipien-der-sicherheitsarchitektur[Prinzipien der Sicherheitsarchitektur] unterschieden, wie das nachfolgende Bild zeigt:
+
+.Authentifizierungswege als Mandanten
+[id="image-authorization-code-connections",reftext="{figure-caption} {counter:figures}"]
+image::konzept/security_multi-tenancy_with_multi_iams.dn.svg[align="center"]
+
+Dabei ergeben sich die unterschiedlichen Kommunikationskanäle _Portal_, _Service-Gateway_, _Authentication-Gateway_ oder _interne Anwendungs-Authentifizierung_ (Batch/Task/Kontextwechsel), über welche ein Authentifizierungs-Request an der Anwendungslandschaft ankommt bzw. verarbeitet wird.
+
+Alle daran beteiligten Komponenten unterscheiden sich in ihrem Aufbau, ihrer Konfiguration und ihrer Funktionalität, so dass jede genau die speziellen Kommunikationskanal-Eigenheiten - siehe dazu Kapitel <<aufruf-weboberflaeche-portal>>, <<aufruf-service-gateway-ohne-authentication-gateway>>, <<aufruf-service-gateway-mit-authentication-gateway>> und <<ausfuehrung-batch-u-task>> bzw. <<aufruf-von-nachbarsystemen>> - am besten bedienen kann.
+Zur Umsetzung und sicherheitsrelevanten Trennung dieser Kommunikationskanäle werden diese als eigenständige Mandanten betrachtet und können somit auch als mehrere Instanzen pro Kanal eingerichtet werden.
```

Comment:
Gilt das 'eigenständige Mandanten' auch für das IAM? (z.B. KeyCloak: werden dort sie in eigene realms abgebildet?) 
Dann sind es logisch gesehen getrennte, eigenständige IAMs, die nur physisch auf einem IAM-System laufen. Die haben unterschiedliche URLs (unterschieden durch realm-Name), die in der Konfiguration (des/der Portale u. Gateways oder Service-anbietende Anwendungen?) über tenants unterschieden werden können.